# the power to be published and send to laser socket
Header header
float32 value
